mock-fedora.cfg

```diff
@@ -18,8 +19,8 @@ config_opts['macros']['clamp_mtime_to_source_date_epoch'] = 'Y'
 config_opts['macros']['use_source_date_epoch_as_buildtime'] = 'Y'
 config_opts['macros']['_buildhost'] = 'reproducible'
 
-config_opts['yum.conf'] = """
-[main]
+config_opts['dnf.conf'] = config_opts['yum.conf'] = """
+[main]{proxy}
```

Comment:
Please add it as a separate line, preferably at the end of the main section. Empty lines are ignored but concatenated like this are unfriendly to anyone reading this file.